code/game/objects/items/oddities.dm

```diff
@@ -158,7 +158,7 @@
 
 /obj/item/weapon/oddity/common/old_money
 	name = "old money"
-	desc = "It's not like organization that issued this exist now."
+	desc = "It's not like organization that issued this exists anymore."
```

Comment:
"It's not like the organization that issued this exists anymore."

Reply:
i missed the "the", oops